test/experimental/features/new_trigger_filters/feature.go

```diff
@@ -202,3 +202,47 @@ func AnyFilterFeature(brokerName string) *feature.Feature {
 
 	return f
 }
+
+func AllFilterFeature(brokerName string) *feature.Feature {
+	f := feature.NewFeature()
+
+	eventContexts := []CloudEventsContext{
+		{
+			eventType:     "exact.prefix.suffix.event.type",
+			shouldDeliver: true,
+		},
+		{
+			eventType:     "not.event.type",
+			shouldDeliver: false,
+		},
```

Comment:
@Leo6Leo could you add events that check for 2/4 filters matching, and 3/4 filters matching and assert that they do not get delivered?